deploy/lvm-operator.yaml

```diff
@@ -109,6 +109,15 @@ spec:
                 - "yes"
                 - "no"
                 type: string
+              thinProvision:
+                description: ThinProvision specifies whether logical volumes can be
+                  thinly provisioned. If it is not set to "yes", then the LVM LocalPV
+                  Driver will create thinProvision i.e. logical volumes that are larger
+                  than the available extents.
```

Comment:
The description is wrong. If `thinProvision` is set to `"yes"`, then then LVM driver will create thin provisioned volumes . .